arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -318,26 +299,52 @@
             "NotSupported"
           ],
           "type": "string",
-          "readOnly": true
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "AutomaticTuningDisabledReason"
+          }
         }
       }
     },
     "AutomaticTuning": {
-      "description": "A list of virtual network rules.",
+      "description": "Database-level Automatic Tuning.",
       "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
       "properties": {
-        "value": {
-          "description": "Array of results.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/VnetFirewallRule"
-          },
-          "readOnly": true
-        },
-        "nextLink": {
-          "description": "Link to retrieve next page of results.",
+        "properties": {
+          "$ref": "#/definitions/AutomaticTuningProperties",
+          "description": "Resource properties.",
```

Comment:
Ok, just wanted to double check to make sure if was a conscious decision on your side.